db/migrate/20190311125542_add_locale_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddLocaleToUsers < ActiveRecord::Migration[5.1]
+  def change
+    add_column :users, :locale, :string, null: false, default: "en"
```

Comment:
Can we use an `integer` and define an `enum` in the user model?
[Help](https://edgeapi.rubyonrails.org/classes/ActiveRecord/Enum.html#method-i-enum)